src/HonzaBotner.Discord.Services/Commands/FunCommands.cs

```diff
@@ -31,13 +30,13 @@ public class FunCommands : ApplicationCommandModule
         }
         Random random = new();
         var text = new StringBuilder("I picked: ");
-        var winNumber = random.Next(answers.Length);
-        var winner = answers[winNumber];
+        int winNumber = random.Next(answers.Length);
+        string winner = answers[winNumber];
```

Comment:
You marked this change as inconsistent in PR #317 ([here](https://github.com/fit-ctu-discord/honza-botner/pull/317#discussion_r964065104))